projects/able-player/template/index.html

```diff
@@ -41,6 +41,10 @@
 
   <body>
     <video data-able-player data-skin="2020" preload="auto" playsinline data-youtube-nocookie data-allow-fullscreen data-youtube-id="<%= ancillary.youtube_url %>">
+    <% if (format.vtt_file) { %>
```

Comment:
i want this to support all the different types of captions. they are listed here https://developer.mozilla.org/en-US/docs/Web/API/VideoTrack/kind

we will need to invent a different field name for each one